staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -253,13 +253,13 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, h *admiss
 	}
 
 	if err := r.Do(ctx).Into(response); err != nil {
-		return false, &webhookutil.ErrCallingWebhook{WebhookName: h.Name, Reason: err}
+		return false, &webhookutil.ErrCallingWebhook{WebhookName: h.Name, Reason: fmt.Errorf("could not send webhook request: %w", err)}
```

Comment:
`failed to call webhook at %s` with the URL

Reply:
-1 on exposing webhook config details to the API caller

Reply:
We expose the webhook URL in almost all other cases:

```
I0427 09:13:00.012155       1 build_controller.go:1779] Giving up retrying skuznets/cli-jq: failed to create build pod: Internal error occurred: failed calling webhook "pod-scaling.ci.openshift.io": Post "https://pod-scaler-consumer-admission.ci.svc:443/?timeout=5s": x509: certificate signed by unknown authority
I0427 15:13:00.100467       1 build_controller.go:1779] Giving up retrying skuznets/cli-jq: failed to create build pod: Internal error occurred: failed calling webhook "pod-scaling.ci.openshift.io": Post "https://pod-scaler-consumer-admission.ci.svc:443/?timeout=5s": no endpoints available for service "pod-scaler-consumer-admission"
I0427 15:28:38.317704       1 build_controller.go:1779] Giving up retrying skuznets/cli-jq: failed to create build pod: Internal error occurred: failed calling webhook "pod-scaling.ci.openshift.io": Post "https://pod-scaler-consumer-admission.ci.svc:443/?timeout=5s": context deadline exceeded
```

Does it matter much @liggitt  ?